assets/js/environment.js

```diff
@@ -215,12 +215,73 @@ function drawVersionChart(stream) {
     },
   }
 
+  async function getData(componentName, previousCommitSha, commitSha) {
+    const url = `https://api.github.com/repos/ministryofjustice/${componentName}/compare/${previousCommitSha}...${commitSha}`
+    try {
+      console.log(url)
+      const githubHeaders = new Headers()
+      githubHeaders.append('Accept', 'application/vnd.github+json')
+      //githubHeaders.append("Authorization", "Bearer xxx")
+      githubHeaders.append('X-GitHub-Api-Version', '2022-11-28')
+
+      const response = await fetch(url, { headers: githubHeaders })
+      if (!response.ok) {
+        throw new Error(`Response status: ${response.status}`)
+      }
+
+      const json = await response.json()
+      return json
+    } catch (error) {
+      console.error(error.message)
```

Comment:
To make it a consistent return, should probably return something `{}` ? when there is an error as you're not actually stopping the code on error here.